developer-docs-site/docs/apis/aptos-labs-developer-portal.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "[Beta] Aptos Labs Developer Portal"
+slug: "aptos-labs-developer-portal"
+---
+
+The [Aptos Labs Developer Portal](https://developers.aptoslabs.com) is a your gateway to access Aptos Labs provided APIs in a quick and easy fashion to power your dApp.
+It consists of a Portal (UI) and a set of API Gateways operated by Aptos Labs.
+
+The Developer Portal aims to make it easier to build dApps by:
+
+1. Providing [unified domain names/URLs](../nodes/networks.md) for each API.
+2. Giving you personalized examples on how to use each API.
+3. Observability into your personal usage, error rates and latency of APIs.
```

Comment:
Description and usage of the aptoslabs developer portal should be on the aptos labs developer portal.

Reply:
deferring to internal discussion we had on this. Seems like this is fine to keep here.